src/components/tiles/custom-editable-tile-title.tsx

```diff
@@ -0,0 +1,97 @@
+import classNames from "classnames";
+import { observer } from "mobx-react";
+import React, { useEffect, useState } from "react";
+import { ExpressionContentModelType } from "../../plugins/expression/expression-content";
+import { DataCardContentModelType } from "../../plugins/data-card/data-card-content";
+
+type SimpleTitleTileTypes = DataCardContentModelType | ExpressionContentModelType;
+
+interface IProps {
+  model: any;
+  onRequestUniqueTitle: any;
+  readOnly: boolean | undefined;
+}
+
+export const CustomEditableTileTitle: React.FC<IProps> = observer((props) => {
```

Comment:
```suggestion
export const CustomEditableTileTitle: React.FC<IProps> = observer(function CustomEditableTileTitle(props) {
```
This way of declaring observing functional components is easier to debug issues with MobX.